sizefashion/src/components/ProductImages.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { getImageTags } from '../utils/helpers';
+
+const ProductImages: React.FC<{ image: string | undefined }> = ({ image = '' }) => {
+  const imageSrc = getImageTags(image)
+
+  return (
+    <div>
+      <img src={imageSrc} alt='main' className='main' />  
```

Comment:
@otaaviostein alt image name is not clear enough. Remember, you're adding an `alt` not only because your compiler complains about it, but because a person that relies on it may read it!